### PR TITLE
Update flow builder actions docs for 6.4

### PR DESCRIPTION
**Pull request by nguyenky**
This is an update of the Flow Builder actions document for `6.4.x` version.
Issue link: https://github.com/shopware/docs/issues/948

### CODE REVIEW

File: guides/plugins/plugins/framework/flow/add-flow-builder-action.md [hunk 1]
```diff
@@ -150,158 +148,198 @@ As you can see, several methods are already implemented:
...
 
-```php
-<?php declare(strict_types=1);
+```xml
+<service id="Swag\CreateTagAction\Core\Content\Flow\Dispatching\Action\CreateTagAction">
+    <argument type="service" id="tag.repository" />
+    <tag name="kernel.event_subscriber"/>
+    <tag name="flow.action" priority="600"/>
+</service>
+```
```
Comment (sushmangupta) [inline guides/plugins/plugins/framework/flow/add-flow-builder-action.md:165]:
```suggestion
```
{%endcode%}
```

File: guides/plugins/plugins/framework/flow/add-flow-builder-action.md [hunk 2]
```diff
@@ -528,56 +613,65 @@ Component.register('swag-example-plugin-modal', {
 
 #### Twig template file
 
-{% code title="<plugin root>/src/Resources/app/administration/src/component/swag-example-plugin-modal/swag-example-plugin-modal.html.twig" %}
+{% code title="<plugin root>/src/Resources/app/administration/src/component/sw-flow-create-tag-modal/sw-flow-create-tag-modal.html.twig" %}
 {% raw %}
```
Comment (sushmangupta) [inline guides/plugins/plugins/framework/flow/add-flow-builder-action.md:617]:
```suggestion
{% code title="<plugin root>/src/Resources/app/administration/src/component/sw-flow-create-tag-modal/sw-flow-create-tag-modal.html.twig" %}
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

File: guides/plugins/plugins/framework/flow/add-flow-builder-action.md [hunk 3]
```diff
@@ -150,158 +148,197 @@ As you can see, several methods are already implemented:
...
+{% code title="<plugin root>/src/Resources/config/services.xml" %}
 
-```php
-<?php declare(strict_types=1);
+```xml
+<service id="Swag\CreateTagAction\Core\Content\Flow\Dispatching\Action\CreateTagAction">
+    <argument type="service" id="tag.repository" />
+    <tag name="kernel.event_subscriber"/>
+    <tag name="flow.action" priority="600"/>
+</service>
```
Comment (sushmangupta) [inline guides/plugins/plugins/framework/flow/add-flow-builder-action.md:164]:
```suggestion
</service>
```

{% endcode %}
```

Comment (sushmangupta) [inline guides/plugins/plugins/framework/flow/add-flow-builder-action.md:164]:
```suggestion
    <tag name="flow.action" priority="600"/>
</service>
```

{% endcode %}
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

File: guides/plugins/plugins/framework/flow/add-flow-builder-action.md (hunk 3 above)
Comment (nguyenky) [reply guides/plugins/plugins/framework/flow/add-flow-builder-action.md:164]: I updated it

**Comment by Isengo1989** [comment]
@sushmangupta tested with 6.4.19.0 - looks good :+1:

**Review by Isengo1989 — APPROVED** [review]

**Review by sushmangupta — APPROVED** [review]
@nguyenky , thanks for updating this info to our docs :)

**Pull request merged**